examples/opentelemetry-tracing/verify.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -e
+
+export NAME=opentelemetry-tracing
+export PORT_PROXY="${PORT_PROXY:-8000}"
+export PORT_COLLECTOR_ZPAGE="${PORT_COLLECTOR_ZPAGE:-55679}"
```

Comment:
the reason for this pattern is to allow the examples to be tested simultaneously (ie to expose different ports when run with verify.sh)

once its clearer what services/ports need to be exposed i can suggest what ports to use here